### PR TITLE
Dev -> Master for 2.1.0 release

### PR DESCRIPTION
**Pull request by Emiller88**

### CODE REVIEW

**Comment by alyssa-ab** [comment]
Everything else looks good to me other than needing a run through black!

File: workflows/nascent.nf [hunk 1]
```diff
@@ -283,12 +283,11 @@ workflow NASCENT {
```
[the remark this answers is not in the document]
Comment (Emiller88) [reply workflows/nascent.nf:283]: #107

File: tests/modules/local/grohmm/parametertuning.nf.test [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                            "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/genome/genome.gtf",
+                            checkIfExists: true
+                            )
+                input[2] = []
+                """
+            }
+        }
+
+        then {
+            // FIXME Broken test
```
[the remark this answers is not in the document]
Comment (Emiller88) [reply tests/modules/local/grohmm/parametertuning.nf.test:30]: Yeah, old pytest workflow tests I converted but I haven't gotten to bottom of it yet. Mainly cause groHMM is a pain.

**Review by alyssa-ab — APPROVED** [review]
LGTM!

**Review by erikrikarddaniel — APPROVED** [review]
Looks good. Ambitious with tests for local modules!

**Pull request merged**